### DISCUSSION TITLE
Delete an element without delete connected link

### DISCUSSION THREAD

**Discussion by malomiquel** [clientIO/joint#1861] [Q&A]
I want to know if it is possible to delete an element where a link is connected without deleting the connected link.

Here is a example:
I want to delete BUS1 without delete the link. For the moment when I delete the BUS1, it delete automatically the link connected.

<img width="260" alt="Capture d’écran 2022-10-25 160845" src="https://user-images.githubusercontent.com/83480447/197796972-b42947aa-2e56-4a80-bed4-a1125a1e474c.png">

**Comment by kumilingus** [comment]
You need to disconnect the connected links first:
https://jsfiddle.net/kumilingus/q8gc7ksh/

You can also call `element.remove({ disconnectLinks: true })`, but this will disconnect links and pin them to point at *0,0*.
